src/components/course-table/course-table.html

```diff
@@ -1,9 +1,10 @@
+<div class="my-container">
 <h1>Courses</h1>
-<h2 v-if="isLoading">Loading...</h2>
-<h2 v-else-if="error">{{ error }}</h2>
+<p v-if="error">{{ error }}</p>
```

Comment:
if you want to show an error then just create a component name 'error' and when you get an error from the store then just toggle its visibility